### PR TITLE
add ansi snapshot test

### PR DESCRIPTION
**Pull request by Gummibeer**

### CODE REVIEW

File: tests/__snapshots__/SnapshotTest__basic_output__1.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
-  GET|HEAD       / ..............................................................
```
Comment (Wulfheart) [inline tests/__snapshots__/SnapshotTest__basic_output__1.txt:1]: Could there be a problem with different terminal sizes when running phpunit from terminal?

File: tests/SnapshotTest.php [hunk 2]
```diff
@@ -7,14 +7,23 @@
 use Illuminate\Support\Facades\Artisan;
 use Spatie\Snapshots\MatchesSnapshots;
 
-class SnapshotTest extends TestCase
+final class SnapshotTest extends TestCase
```
Comment (Wulfheart) [inline tests/SnapshotTest.php:10]: Any specific reason for using `final`on this? I wonder what the benefits are.

**Review by Wulfheart — COMMENTED** [review]
Thank you for submitting this.

File: tests/__snapshots__/SnapshotTest__basic_output__1.txt (hunk 1 above)
Comment (Gummibeer) [reply tests/__snapshots__/SnapshotTest__basic_output__1.txt:1]: Not anymore - it's forced to 120 columns now.

File: tests/SnapshotTest.php (hunk 2 above)
Comment (Gummibeer) [reply tests/SnapshotTest.php:10]: I always use final on tests as there's absolutely no need to extend a test class.

File: tests/TestCase.php [hunk 3]
```diff
@@ -2,44 +2,23 @@
...
-            'driver' => 'sqlite',
-            'database' => ':memory:',
-            'prefix' => '',
-        ]);
-
-        /*
-        include_once __DIR__.'/../database/migrations/create_pretty_routes_table.php.stub';
-        (new \CreatePackageTable())->up();
-        */
+        putenv('COLUMNS=120');
```
Comment (Gummibeer) [inline tests/TestCase.php:18]: Here we define a fixed terminal width now for all tests.

Comment (Wulfheart) [reply tests/TestCase.php:18]: Nice!

File: tests/SnapshotTest.php (hunk 2 above)
Comment (Wulfheart) [reply tests/SnapshotTest.php:10]: Ok.

**Pull request merged**